Unix/configeditor/configeditor.cpp

```diff
@@ -423,13 +446,30 @@ int MI_MAIN_CALL main(int argc, const char** argv)
             }
             ss << commentByte << prop << "=" << _CreateStringFromList(valueString,valueList);
         }
+        // Support querying a value from a property list
+        else if (!opts.query.empty())
+        {
+        	std::list<std::string> valueList;
+        	
```

Comment:
Needless spaces in here. Blank line is fine (and good), prefer blank line without spaces.